apps/storybook/.storybook/main.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+
+module.exports = {
+  stories: ['../src/**/*.stories.@(tsx|mdx)'],
+  addons: [
+    '@storybook/preset-create-react-app',
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+  ],
+  webpackFinal: (config) => {
+    // Let webpack process raw TS files imported from local packages
+    config.module.rules[2].oneOf[2].include.push(
```

Comment:
Yeah, since it's not CRA, I can't use the utility from `react-app-rewire-alias`... I think it's better to have something very targetted and clear rather than convoluted and hard to debug. When we upgrade Storybook, if this breaks, at least we'll know it's just because the position of the rule changed, not because the code we use to find it uses the wrong regex or whatever...